tests/src/JIT/SIMD/Vector3TestNative.cpp

```diff
@@ -21,6 +21,10 @@
 #endif  // !__i386__
 #endif // !defined(_MSC_VER)
 
+#ifndef _countof
```

Comment:
It was previously suggested that this test should be moved to the Interop tests, but we felt that it belongs with the SIMD tests. I agree that it would be a good thing to have this is a common location.